modules/distr-tracing-removing-instance-cli.adoc

```diff
@@ -0,0 +1,90 @@
+////
+This module included in the following assemblies:
+- distr_tracing_install/dist-tracing-removing.adoc
+////
+
+[id="dist-tracing-removing-cli_{context}"]
+= Removing a {JaegerName} instance from the CLI
+
+. Log in to the {product-title} CLI.
++
+[source,terminal]
+----
+$ oc login
+----
++
+. To display the {JaegerShortName} instances run the command:
++
+[source,terminal]
+----
+$ oc get deployments -n <jaeger-project>
+----
++
+For example,
++
+[source,terminal]
+----
+$ oc get deployments -n openshift-operators
+----
++
+The names of Operators have the suffix `-operator`. The following example shows two {JaegerName} Operators and four {JaegerShortName} instances:
++
+[source,terminal]
+----
+$ oc get deployments -n openshift-operators
+----
++
+You should see output similar to the following:
++
+[source,terminal]
+----
+NAME                     READY   UP-TO-DATE   AVAILABLE   AGE
+elasticsearch-operator   1/1     1            1           93m
+jaeger-operator          1/1     1            1           49m
+jaeger-test              1/1     1            1           7m23s
+jaeger-test2             1/1     1            1           6m48s
+tracing1                 1/1     1            1           7m8s
+tracing2                 1/1     1            1           35m
+----
++
+. To remove an instance of {JaegerShortName}, run the command:
++
+[source,terminal]
+----
+$ oc delete jaeger <deployment-name> -n <jaeger-project>
+----
++
+For example,
++
+[source,terminal]
+----
+$ oc delete jaeger tracing2 -n openshift-operators
+----
++
+
+. To verify the deletion, run `oc get deployment` again:
```

Comment:
```suggestion
. To verify the deletion, run the `oc get deployments` subcommand again:
```
For consistency, it would help to use either singular or plural "oc get deployments" in each instance of this procedure to avoid confusion